app/src/main/java/org/p2p/wallet/send/interactor/SendInteractor.kt

```diff
@@ -65,24 +70,24 @@ class SendInteractor(
 
     // Fees calculator
     suspend fun calculateFeesForFeeRelayer(
-        token: Token.Active,
-        receiver: String?,
+        sourceToken: Token.Active,
+        receiverAddress: String?,
         networkType: NetworkType
     ): FeeRelayerSendFee? {
-
         return when (networkType) {
-            NetworkType.BITCOIN ->
+            NetworkType.BITCOIN -> {
                 FeeRelayerSendFee(
-                    feeInSol = BigInteger.valueOf(20000L),
-                    feeInPayingToken = null
+                    solFee = AmountInLamports(BigInteger.valueOf(20000L)),
```

Comment:
to const do not know name =)
BITCOIN_FEE = 20000L
solFee = AmountInLamports(BITCOIN_FEE.toBigInteger()),